src/test/java/Homework17.java

```diff
@@ -0,0 +1,74 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class Homework17 extends BaseTest{
+
+
+    WebDriver driver;
+
+    @BeforeMethod
+    public void beforeMethod(){
+//        driver = new ChromeDriver();    // set driver
+        driver = new FirefoxDriver();
+        KoelTesting.goToKoel(driver);
+    }
```

Comment:
Base Page \ Base Class — Implements the necessary methods for work with webdriver.

_For example_

![image](https://user-images.githubusercontent.com/104761482/210751389-73ee1f1c-fc2d-4705-a98d-095a364e5d3c.png)